README.md

```diff
@@ -61,70 +61,46 @@ Then manually install the following JARs:
 
 ## Example
 
-list all pods:
-
-```java
```

Comment:
Let's leave the simple code example in also, in addition to the list of examples you added.

Reply:
i kept the simple example and added that to the list. wdyt?

Reply:
I feel like if I maven import this library, having to do an extra click to get an example is too much work. I'd still like to include both.